src/guc.c

```diff
@@ -95,6 +95,8 @@ bool ts_shutdown_bgw = false;
 char *ts_current_timestamp_mock = NULL;
 #endif
 
+int ts_guc_debug_toast_tuple_target = 128;
+
```

Comment:
do we really need a guc for this for testing, we can just set the corresponding option directly in tests

Reply:
It worked when we had inheritance, but now the compressed chunk is re-created at compression w/o inheritance, so I'm not sure it's even possible.

Reply:
The GUC is debug-only, in release builds it's just a static variable that is never changed.